### PR TITLE
Add specific instructions for installing Rust in Windows

### PR DESCRIPTION
**Pull request by maurolacy**
Related to [cosmwasm #630](https://github.com/CosmWasm/cosmwasm/issues/630).

### CODE REVIEW

**Review by orkunkl — APPROVED** [review]
Nice one LGTM! @webmaster128 should take a look

File: getting-started/installation.md [hunk 1]
```diff
@@ -31,6 +33,36 @@ rustup target list --installed
...
+If requested, manually download and install Visual C++ Build Tools 2019,
+from https://visualstudio.microsoft.com/visual-cpp-build-tools/ .
+Make sure "Windows 10 SDK" and "English language pack" are selected.
+
+Continue running `rustup-init.exe`, and proceed with the installation.
+
+Optionally:
+- Download and install [gvim](https://www.vim.org/download.php#pc), and modify the Env vars to add \<gvim folder\>
+to the PATH.
+- Download and install [git for windows](https://git-scm.com/download/win). Modify the Env vars to add <git folder>\bin
```
Comment (webmaster128) [inline getting-started/installation.md]: Does this need escaping too?

File: getting-started/installation.md [hunk 2]
```diff
@@ -31,6 +33,36 @@ rustup target list --installed
...
+to PATH.
+- Turn on Developer Mode (Settings -> Update and Security: For Developers) and enable Device Discovery, to be able to
+access the Windows 10 server through ssh (https://www.ctrl.blog/entry/how-to-win10-ssh-service.html#section-mssshserv-enable).
+
+Install the wasm32 target:
+```shell
+rustup default stable
+cargo version
+# If this is lower than 1.44.1+, update
```
Comment (webmaster128) [inline getting-started/installation.md]:
```suggestion
# If this is lower than 1.47.0, update
```

**Review by webmaster128 — COMMENTED** [review]
Nice

**Pull request merged**